package.json

```diff
@@ -243,7 +243,7 @@
 		"build:plugin-zip": "bash ./bin/build-plugin-zip.sh",
 		"build": "npm run build:packages && wp-scripts build",
 		"changelog": "node ./bin/plugin/cli.js changelog",
-		"check-licenses": "concurrently \"wp-scripts check-licenses --prod --gpl2 --ignore=@react-native-community/cli,@react-native-community/cli-platform-ios\" \"wp-scripts check-licenses --dev\"",
+		"check-licenses": "concurrently \"wp-scripts check-licenses --prod --gpl2 --ignore=@juggle/resize-observer,@react-native-community/cli,@react-native-community/cli-platform-ios\" \"wp-scripts check-licenses --dev\"",
```

Comment:
`@juggle/resize-observer` is licensed as `Apache-2.0` that isn't GPL2 compatible and therefore can't be used in the production code.

Reply:
Yes, I can confirm that @youknowriad is correct about the usage of the polyfill in https://github.com/WordPress/gutenberg/pull/40509#issuecomment-1105080827. 

There is no usage in the bundle shipped with the package:
https://unpkg.com/browse/use-resize-observer@8.0.0/dist/bundle.esm.js

We can always fork this simple utility to avoid future issues with the problematic polyfill.

Reply:
I'd be OK with forking into Gutenberg too.

Reply:
What's the browser support situation?

Reply:
So what do we do here? Just merge or copy the code?